ext/openssl/ossl_ssl.c

```diff
@@ -1046,6 +1046,34 @@ ossl_sslctx_set_ciphers(VALUE self, VALUE v)
     return v;
 }
 
+/*
+ * call-seq:
+ *    ctx.enable_fallback_scsv() => nil
+ *
+ * Activate TLS_FALLBACK_SCSV for this context.
+ * See RFC 7507.
+ */
+static VALUE
+ossl_sslctx_enable_fallback_scsv(VALUE self)
+{
+#ifdef SSL_MODE_SEND_FALLBACK_SCSV
```

Comment:
The method should not be defined at all if SSL_MODE_SEND_FALLBACK_SCSV is not usable.

Reply:
Fixed